python/ql/src/Security/CWE-295/MissingHostKeyValidation.qhelp

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+  <p>
+In the Secure Shell (SSH) protocol, host keys are used to verify the identity of
+remote hosts. Accepting unknown host keys may leave the connection open to
+man-in-the-middle attacks.
+</p>
+</overview>
+
+<recommendation>
+<p>
+Do not accept unknown host keys. For the Paramiko library in particular, avoid
+setting the missing host key policy to either <code>AutoAddPolicy</code> or
+<code>WarningPolicy</code>, as both of these will continue even when the host
+key is unknown. The default <code>RejectPolicy</code> throws an exception when
+unknown host keys are encountered.
```

Comment:
Suggest something more like this?

"Do not accept unknown host keys. In particular, do not set the default missing host key policy for the Paramiko library to either <code>AutoAddPolicy</code> or <code>WarningPolicy</code>. Both of these policies continue even when the host key is unknown. The default setting of <code>RejectPolicy</code> is secure because it throws an exception when it encounters an unknown host key. "